src/main/java/uk/gov/ons/ctp/common/error/RestExceptionHandler.java

```diff
@@ -34,8 +33,9 @@
    */
   @ExceptionHandler(CTPException.class)
   public ResponseEntity<?> handleCTPException(CTPException exception) {
-    log.error("handleCTPException");
-    log.error(exception.toString(), exception);
+    log.with("fault", exception.getFault())
+        .with("exception_message", exception.getMessage())
+        .error(exception, "Uncaught CTPException");
```

Comment:
isn't it more common to do this the other way around (i.e. message first, then the exception)?